### PR TITLE
Fixes #4020 - Move static handling to a filter

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Needed for @static to work on other OSes

### CODE REVIEW

File: app/controllers/unattended_controller.rb [hunk 1]
```diff
@@ -220,6 +219,11 @@ def waik_attributes
 
   private
 
+  def set_static
+    # force static network configuration if static http parameter is defined, in the future this needs to go into the GUI
+    @static = !params[:static].empty?
```
Comment (domcleal) [inline app/controllers/unattended_controller.rb]: Could it just move into load_template_vars?

Comment (GregSutcliffe) [inline app/controllers/unattended_controller.rb]: Good catch, done.

**Comment by domcleal** [comment]
Could you add a functional test please?  Looks like we're missing coverage of `@static`.

**Comment by GregSutcliffe** [comment]
@domcleal done

**Comment by domcleal** [comment]
@GregSutcliffe they broke :(

**Comment by GregSutcliffe** [comment]
@domcleal bugger, I can't replicate that at all...

**Comment by GregSutcliffe** [comment]
Ok, I can work around it by disabling Safemode for that test (as currently pushed), will that do?

**Comment by domcleal** [comment]
Yup.  Thanks, merged as 51e8d9d for 1.4.

**Comment by ares** [comment]
oops, empty "false," file got in :-) unless anyone else sends PR before me, I'll remove it in first PR, I'll send

**Comment by domcleal** [comment]
Argh, thanks @ares, I meant to do that.  Will remove it now.

**Pull request closed without merging**